src/sentry/static/sentry/app/api.jsx

```diff
@@ -259,7 +266,10 @@ export class Client {
             ...args
           );
         },
-        complete: this.wrapCallback(id, options.complete, true),
+        complete: () => {
+          Sentry.getCurrentHub().finishSpan(requestSpan);
```

Comment:
Should we take the hub from the `Sentry.getCurrentHub()` above (line 183) and pass the reference here instead of this getCurrentHub again? Considering the _current hub_ could've changed, for example.